zowe/zowe-cli
# Only one error format for V3 & remove references to bright

**What It Does**


The revised format of Zowe CLI error messages (which was an option controlled by the environment variable ZOWE_V3_ERR_FORMAT during V2) is the only error format used by Zowe CLI in V3.

Also remove all references to 'bright' and 'brightside' within the app.

Closes [#1792](https://github.com/zowe/zowe-cli/issues/1792)

**How to Test**


You can test that the new error format is automatically being used by issuing the following erroneous command.

```
zowe zos-files list all-members "DATASET.DOES.NOT.EXIST"
```
The output will begin with the text:

```
Unable to perform this operation due to the following problem.
ISRZ002 Data set not cataloged - 'DATASET.DOES.NOT.EXIST' was not found in catalog.
```

You can confirm that the ```bright``` command is gone by installing zowe-cli into a clean environment. You then will get an error like the following when using the ```bright``` command.

```
> bright --version
bright: The term 'bright' is not recognized as a name of a cmdlet, function, script file, or executable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
```

**Review Checklist**
I certify that I have:


**Additional Comments**
